CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
 # Cromwell Change Log
 
 ## 0.20
+
+* The default per-upload bytes size for GCS is now the minumum 256K
+instead of 64M. There is also an undocumented config key
```

Comment:
Is it undocumented on purpose ? If we want to make it configurable (which I'm not sure we need to) then we should drop a line in the README IMO.